src/cromshell/logs/command.py

```diff
@@ -0,0 +1,267 @@
+import logging
+
+import click
+import gcsfs
+from termcolor import colored
+
+from cromshell.metadata import command as metadata_command
+from cromshell.utilities import io_utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="logs")
+@click.argument("workflow_id")
+@click.option(
+    "-s",
+    "--status",
+    default="Failed",
+    help="Return a list with links to the logs with the indicated status. "
+    "Separate multiple keys by comma or use 'ALL' to print all logs. "
+    "Some standard Cromwell status options are 'ALL', 'Done', 'RetryableFailure', 'Running', and 'Failed'.",
+)
+@click.option(
+    "-p",
+    "--print-logs",
+    is_flag=True,
+    default=False,
+    help="Print the contents of the logs to stdout if true. "
+    "Note: This assumes GCS bucket logs with default permissions otherwise this may not work",
+)
+@click.option(
+    "-des",
+    "--dont-expand-subworkflows",
+    is_flag=True,
+    default=False,
+    help="Do not expand subworkflow info in metadata",
+)
+@click.pass_obj
+def main(
+    config,
+    workflow_id: str,
+    status: list,
+    dont_expand_subworkflows: bool,
+    print_logs: bool,
+):
+    """Get a subset of the workflow metadata."""
+
+    LOGGER.info("logs")
+
+    # If no keys were provided then set key_param to empty else
+    # strip trailing comma from keys and split keys by comma
+    status_param = (
+        ["ALL"]
+        if "ALL".lower() in status.lower()
+        else str(status).strip(",").split(",")
```

Comment:
might be worth being clever and making these case insensitive by manually matching/converting to cromwell values.  Might not be though...